### DISCUSSION TITLE
How can I have grouped columns on the table?

### DISCUSSION THREAD

**Discussion by maralihart** [microsoft/fluentui#30456] [Q&A]
![image](https://github.com/microsoft/fluentui/assets/42477471/745c152c-84b0-4492-8d05-a0b7bc3a78a6)

How could I make grouped columns like Column A, B, and C, which rest on top of another header?

**Comment by tombohub** [comment]
Hi, this doesn't seem to be supported out of the box. Did you try anything so far?

**Reply by maralihart** [reply]
I couldn't think of anything and was advised by someone to just do a regular HTML table because they don't have a `colspan` attribute.

**Reply by tombohub** [reply]
I am not sure myself either. I guess you can use `noNativeElements` prop for the table which will render table as flex divs instead of html table maybe you can attach another header on top? Or you can use tanstack table and style it with fluent ui tokens?

**Comment by maralihart** [comment]
I just ended up using HTML table tags and styling with fluent UI. Looks
pretty similar and I didn’t need sorting or resizing anyway.

**Reply by tombohub** [reply]
nice, maybe you can share code to spread the knowledge. Probably somebody will want similar in the future
